bin/^3.3.x-dev/prepare_project_edition.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
```

Comment:
This file has been copied from https://github.com/ibexa/ci-scripts/blob/0.1/bin/prepare_project_edition.sh